netbox/ipam/views.py

```diff
@@ -547,10 +548,42 @@ def get(self, request, pk):
             'ipaddress': ipaddress,
             'parent_prefixes_table': parent_prefixes_table,
             'duplicate_ips_table': duplicate_ips_table,
+            'more_duplicate_ips': duplicate_ips.count() > 10,
             'related_ips_table': related_ips_table,
         })
 
 
+class IPAddressDuplicatesView(ObjectListView):
```

Comment:
We shouldn't need a separate view for this. Just linking to the IP addresses list with the appropriate `address` and `vrf_id` filters should suffice.

Reply:
🤦 I don't know why I didn't think of that. Will do.